### PR TITLE
PP-9004 Check org details page

### PR DESCRIPTION
**Pull request by iqbalgds**
- Add path, route, controller for the new `Check org details` page.

### CODE REVIEW

File: app/paths.js [hunk 1]
```diff
@@ -22,7 +22,8 @@ module.exports = {
     },
     credentials: {
       index: '/your-psp/:credentialId/credentials',
-      edit: '/your-psp/:credentialId/credentials/edit'
+      edit: '/your-psp/:credentialId/credentials/edit',
+      checkOrgDetails: '/your-psp/:credentialId/credentials/check-organisation-details'
```
Comment (kbottla) [inline app/paths.js:26]: Can this be moved to yourPSP->stripeSetup as this is part of stripe onboarding? 

Also the url should be `/your-psp/:credentialId/check-organisation-details`

File: app/controllers/credentials.controller.js [hunk 2]
```diff
@@ -187,9 +187,14 @@ async function updateNotificationCredentials (req, res, next) {
   }
 }
 
+function checkOrgDetails (req, res, next) {
```
Comment (kbottla) [inline app/controllers/credentials.controller.js:190]: This should be under stripe-setup as check organisation details is part of stripe onboarding

File: app/routes.js [hunk 3]
```diff
@@ -335,6 +335,8 @@ module.exports.bind = function (app) {
   account.get(notificationCredentials.edit, permission('gateway-credentials:update'), credentialsController.editNotificationCredentials)
   account.post(notificationCredentials.update, permission('gateway-credentials:update'), credentialsController.updateNotificationCredentials)
 
+  account.get(credentials.checkOrgDetails, permission('gateway-credentials:read'), credentialsController.checkOrgDetails)
```
Comment (kbottla) [inline app/routes.js:338]:
```suggestion
  account.get(yourPsp.stripeSetup.checkOrgDetails, permission('stripe-organisation-details:update),  ...)
```

**Review by kbottla — CHANGES_REQUESTED** [review]

File: app/paths.js (hunk 1 above)
Comment (iqbalgds) [reply app/paths.js:26]: Thank - will update that

File: app/controllers/credentials.controller.js (hunk 2 above)
Comment (iqbalgds) [reply app/controllers/credentials.controller.js:190]: Thanks - created a new controller

File: app/routes.js (hunk 3 above)
Comment (iqbalgds) [reply app/routes.js]: Thanks - have updated it.

File: app/views/credentials/check-org-details.njk [hunk 4]
```diff
@@ -0,0 +1,62 @@
+{% extends "layout.njk" %}
```
[the remark this answers is not in the document]
Comment (iqbalgds) [reply app/views/credentials/check-org-details.njk]: Done - thanks

**Comment by iqbalgds** [comment]
Update this so this page can be accessed from 3 paths as per the pages:
- KYC
- New Stripe setup
- Switch PSP

**Review by kbottla — CHANGES_REQUESTED** [review]

**Review by kbottla — APPROVED** [review]

**Pull request merged**